recipes/glimpse-bio/build.sh

```diff
@@ -1,22 +1,27 @@
 #!/bin/bash
 
+export __COMMIT_ID__="$(git rev-parse --short HEAD)"
+export __COMMIT_DATE__="$(git show -s --format="%ci" | cut -f 1 -d ' ')"
```

Comment:
I don't think we can use `git` commands here, since it's currently not in a git repo at this point.  I think the best we can do is below:

```suggestion
export __COMMIT_ID__="${GIT_FULL_HASH}"
export __COMMIT_DATE__="$(date -Idate -u)"
```

Reply:
That's unfortunate effectively.
It would have been nice!

Reply:
Unfortunately:
`export __COMMIT_ID__="${GIT_FULL_HASH}"`
Doesn't seems to work.